display_list/dl_paint.h

```diff
@@ -66,6 +66,20 @@ class DlPaint {
     return *this;
   }
 
+  // In the Impeller backend, we'll *always* apply dithering to gradients, and
+  // *never* apply dithering otherwise, and there will be no user-facing feature
+  // to change this behavior.
+  //
+  // To temporarily match this behavior in the Skia backend, we tag gradients
+  // with a special flag, and then check for that flag when converting from
+  // DlPaint to SkPaint.
+  //
+  // TODO(matanl): Remove this flag when the Skia backend is removed,
```

Comment:
Can't this be removed now? This method isn't called any more.

Reply:
Done.